### PR TITLE
Test launcher mac menu

### PR DESCRIPTION
**Pull request by Siedlerchr**
- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by Siedlerchr** [comment]
grafik

menu works but still showing the wrong program info

File: src/main/java/org/jabref/cli/JabRefLauncher.java [hunk 1]
```diff
@@ -74,7 +78,7 @@ public static void main(String[] args) {
                     return;
                 }
 
-                MainApplication.create(argumentProcessor.getParserResults(), argumentProcessor.isBlank(), preferences);
+               new MainApplication().main(argumentProcessor.getParserResults(), argumentProcessor.isBlank(), preferences, arguments);
```
Comment (tobiasdiez) [inline src/main/java/org/jabref/cli/JabRefLauncher.java:81]: `new` shouldn't be necessary here

**Comment by koppor** [comment]
MWE code for a JavaFX application https://github.com/dlemmermann/JPackageScriptFX/blob/master/jpackagefx-main/src/main/java/com/dlsc/jpackagefx/App.java - seems to be similar to the current one. In yesterday's call we guessed, it could be a timing issue.

**Comment by Siedlerchr** [comment]
Fuu. Now it doesn't event start when I download the build version

**Comment by tobiasdiez** [comment]
Can you please set a breakpoint in https://github.com/openjdk/jfx/blob/b3eec1d655ca8d17d13615ea430e92058d99ec1c/modules/javafx.graphics/src/main/java/com/sun/javafx/application/LauncherImpl.java#L675, and report the differences between the current vs old startup. Thanks!

**Comment by Siedlerchr** [comment]
Okay, now it seems to work fine for me.  But normally when I run the dmg I get the security dialog. I now had to open it with the commandline first to trigger the security dialog. I will need to test if this works with signing 

grafik

**Comment by Siedlerchr** [comment]
This seems to work now. I will check again with the master then

File: src/main/java/org/jabref/cli/Launcher.java [hunk 2]
```diff
@@ -45,8 +45,10 @@
  */
 public class Launcher {
     private static Logger LOGGER;
+    private static String[] ARGUMENTS;
```
Comment (tobiasdiez) [inline src/main/java/org/jabref/cli/Launcher.java:48]: Is this field really necessary?

**Review by tobiasdiez — COMMENTED** [review]
Thanks for fixing this!

File: src/main/java/org/jabref/cli/Launcher.java (hunk 2 above)
Comment (Siedlerchr) [reply src/main/java/org/jabref/cli/Launcher.java:48]: Just to make sure the original args are no modified

**Pull request merged**